src/components/oppfolgingsoppgave/OppfolgingsoppgaveModal.tsx

```diff
@@ -119,10 +145,23 @@ export const OppfolgingsoppgaveModal = ({ isOpen, toggleOpen }: Props) => {
               </option>
             ))}
           </Select>
+          {isBeskrivelseInputVisible && (
+            <Textarea
+              label={texts.beskrivelseLabel}
+              size="small"
+              value={watch("beskrivelse")}
+              maxLength={MAX_LENGTH_BESKRIVELSE}
+              error={errors.beskrivelse && texts.errorTooLongBeskrivelse}
```

Comment:
Tror egentig du kan droppe error prop for `maxLength`. Det er innebygd i Textarea slik at det blir dobbel feilmelding hvis vi har en egen i tillegg:
![image](https://github.com/navikt/syfomodiaperson/assets/79838644/b6904a0a-1f79-4c78-8ecb-2d8f66b9673f)